### ISSUE TITLE
[hecs-plugin-three] Be able to set WebGLRenderer constructor params

### ISSUE THREAD

**Issue by canadaduane** [gohyperr/hecs#23]
Currently, there is no way to configure the renderer created in Presentation:

```
  createRenderer() {
    const renderer = new THREE.WebGLRenderer({
      antialias: true,
    })
...
```

As far as I can tell, these constructor params cannot be set after-the-fact.

**Use Case:** In order to get mixed-mode CSS3D+WebGL working, I need the `alpha: true` setting for WebGLRenderer.

**Comment by ashconnell** [comment]
Yeah for now it's just using some hardcoded defaults. It could probably have a method `world.presentation.configure(options)` to rebuild the WebGLRenderer/Camera/Scene etc based on some options if needed

**Comment by canadaduane** [comment]
Oh, that's an interesting approach. So basically throw away what was already configured, and build it again. You get the advantage of reasonable "default settings" as well as a way to customize settings.

This might also be a good way to handle the default lighting situation.

**Comment by ashconnell** [comment]
I'd be happy to update `hecs-plugin-three` to support replacing the scene, camera, renderer etc using this flow https://github.com/gohyperr/hecs/pull/26#issuecomment-730060336

**Comment by canadaduane** [comment]
Yes, that would be perfect.
